<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kinematic Design 2D</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Kinematics</string>
    </property>
    <addaction name="actionCalculateSolution4RLinkage"/>
    <addaction name="actionCalculateSolutionSliderCrank"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionRunBackward"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionStepForward"/>
    <addaction name="actionStepBackward"/>
    <addaction name="separator"/>
    <addaction name="actionCollisionCheck"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QMenu" name="menuFixed_Body">
     <property name="title">
      <string>Fixed Body</string>
     </property>
     <addaction name="actionFixedRectangle"/>
     <addaction name="actionFixedCircle"/>
     <addaction name="actionFixedPolygon"/>
    </widget>
    <widget class="QMenu" name="menuMoving_Body">
     <property name="title">
      <string>Moving Body</string>
     </property>
     <addaction name="actionMovingRectangle"/>
     <addaction name="actionMovingCircle"/>
     <addaction name="actionMovingPolygon"/>
    </widget>
    <addaction name="actionSelect"/>
    <addaction name="menuFixed_Body"/>
    <addaction name="menuMoving_Body"/>
    <addaction name="separator"/>
    <addaction name="actionLinkageRegion"/>
    <addaction name="actionLinkageAvoidanceRegion"/>
    <addaction name="separator"/>
    <addaction name="actionKinematics"/>
   </widget>
   <widget class="QMenu" name="menuLayer">
    <property name="title">
     <string>Layer</string>
    </property>
    <addaction name="actionAddLayer"/>
    <addaction name="actionInsertLayer"/>
    <addaction name="actionDeleteLayer"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionCircularRepeat"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMode"/>
   <addaction name="menuLayer"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStepForward">
   <property name="text">
    <string>Step Forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionStepBackward">
   <property name="text">
    <string>Step Backward</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionShowCenterPointCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Center Point Curve</string>
   </property>
  </action>
  <action name="actionShowCirclePointCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Circle Point Curve</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionFixedRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionFixedPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionLayer1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Layer 1</string>
   </property>
  </action>
  <action name="actionLayer2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Layer 2</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionKinematics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kinematics</string>
   </property>
  </action>
  <action name="actionLayer3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Layer 3</string>
   </property>
  </action>
  <action name="actionCalculateSolution4RLinkage">
   <property name="text">
    <string>Calculate Solution for 4R Linkage</string>
   </property>
  </action>
  <action name="actionCollisionCheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Collision Check</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRunBackward">
   <property name="text">
    <string>Run Backward</string>
   </property>
  </action>
  <action name="actionFixedCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionLinkageRegion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linkage Region</string>
   </property>
  </action>
  <action name="actionLinkageAvoidance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linkage Avoidance</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionCircularRepeat">
   <property name="text">
    <string>Circular Repeat</string>
   </property>
  </action>
  <action name="actionCalculateSolutionSliderCrank">
   <property name="text">
    <string>Calculate Solutions for Slider Crank</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="text">
    <string>Add Layer</string>
   </property>
  </action>
  <action name="actionInsertLayer">
   <property name="text">
    <string>Insert Layer</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionDeleteLayer">
   <property name="text">
    <string>Delete Layer</string>
   </property>
  </action>
  <action name="actionRectangle_2">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionMovingRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionMovingCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionMovingPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionLinkageAvoidanceRegion">
   <property name="text">
    <string>Linkage Avoidance Region</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
